library/sass/_setup.scss

```diff
@@ -68,13 +69,8 @@ $font-weight-bold: 900;
 
 /*** Thick Borders ***/
 
-@mixin border($direction:null,$color:$blue) {
-  @if ($direction == 'top') {
-    border-top: 6px solid;
-  }
-  @if ($direction == 'bottom') {
-    border-bottom: 6px solid;
-  }
+@mixin border($direction:null,$color:$blue-border) {
+  border-#{$direction}: 5px solid;
```

Comment:
I think this line needs an `@if`, because `$direction` is initially set to `null`. if in our usage we don't specify `$direction` (e.g. `@include border();`) it would break right?

Reply:
Correct, good catch. Instead of an @if, I will add a `!default` value of `top`. Still good to pass a value of your own through, even if it is `top` for readability when used.

Reply:
`!default` is a good idea that I hadn't thought of! and yeah I agree that expanding the mixin for different border directions is interesting but seems like overkill.